app/src/main/java/app/revanced/manager/ui/viewmodel/MainViewModel.kt

```diff
@@ -52,7 +53,10 @@ class MainViewModel(
     }
 
     private suspend fun checkForManagerUpdates() {
-        if (!prefs.managerAutoUpdates.get() || !networkInfo.isConnected()) return
+        if (BuildConfig.DEBUG || 
+            BuildConfig.VERSION_NAME.contains("-dev") || 
+            !prefs.managerAutoUpdates.get() || 
+            !networkInfo.isConnected()) return
```

Comment:
We have `https://github.com/ReVanced/revanced-manager/blob/39536c0e18cc7ffac08e2160e352e35104978b75/app/src/main/java/app/revanced/manager/util/Util.kt#L48` for this purpose